typespec/docs/endpoints.tsp

```diff
@@ -42,6 +42,18 @@ interface SearchEndpoints<DocString extends valueof string = "Search for documen
     ): DocSearchResponse;
 }
 
+interface BulkDeleteEndpoints<DocString extends valueof string = "Bulk delete documents"> {
+    @doc(DocString)
+    @delete
+    deleteAll(
+        @path
+        @doc("ID of the parent")
+        id: uuid,
+
+        body: BulkDeleteDocsRequest,
```

Comment:
damn good catch by @ellipsis-dev actually. this should be `...BulkDeleteDocsRequest` instead of `body: BulkDeleteDocsRequest`